VideoWeb/VideoWeb/Controllers/ConferenceManagementController.cs

```diff
@@ -185,6 +185,15 @@ public async Task<IActionResult> DismissWitnessAsync(Guid conferenceId, Guid par
                     Participant_id = participantId,
                     Transfer_type = TransferType.Dismiss
                 });
+
+                _logger.LogDebug("Sending alert to vho witness {Participant} dismissed from video hearing {Conference}",
+                    participantId, conferenceId);
+                await _videoApiClient.AddTaskAsync(conferenceId, new AddTaskRequest 
+                { 
+                    Participant_id = participantId, 
+                    Body = "Witness dismissed",
+                    Task_type = TaskType.Participant 
+                });
                 return Accepted();
             }
             catch (VideoApiException ex)
```

Comment:
Can you also add a log error in the catch block too please

Reply:
updated to a try catch and added logging for the error